src/pages/CompanyFinancial.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getFinancial } from '../redux/CompanyFinancialSlice';
+import Company from '../components/singlecompany';
+import Header from './Navbar';
+
+const CompanyFinancial = () => {
+  const { financialstats } = useSelector((state) => state.finance);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getFinancial());
+  }, [dispatch]);
+
+  return (
+    <>
+      <Header />
+      {financialstats.map((comp) => (
+        <Company
+          key={comp.id}
+          id={comp.id}
+          companySymbol={comp.companySymbol}
+          reportedCurrency={comp.reportedCurrency}
+          calendarYear={comp.calendarYear}
+          netIncome={comp.netIncome}
+        />
+      ))}
+    </>
+  );
```

Comment:
- [ ] [ optional ] the app is not responsive on-screen mobiles it would be better to make the app responsive on all mobile screens in order to ensure a great UX (user experience)

Reply:
hello @youmari  could you clarify more on the design. i find difficulty understanding the filter question

Reply:
@hakichris, for example, in a search input when I enter the name of the item it should filter the list of the items based on the name that I enter like if we have a list of countries and I enter India in the search input.  the list should be filtered and only India should be displayed

Reply:
tthank you, this is resolved.